common/component/azure/servicebus/metadata.go

```diff
@@ -188,6 +189,17 @@ func ParseMetadata(md map[string]string, logger logger.Logger, mode byte) (m *Me
 		return m, err
 	}
 
+	if strings.Contains(strings.ToLower(m.ConnectionString), "usedevelopmentemulator=true") {
+		if !m.DisableEntityManagement {
+			logger.Warn(
+				"UseDevelopmentEmulator=true detected in connection string. " +
+					"Azure emulator does not support topic management APIs. " +
+					"Dapr will skip admin operations. " +
+					"To suppress this warning, explicitly set disableEntityManagement: true.")
+		}
+		m.DisableEntityManagement = true
```

Comment:
This isn’t my area of expertise, but I want to challenge whether this is really needed given that users can already set `disableEntityManagement=true`